src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -427,6 +440,7 @@ export async function getPriceQuoteLegacy(params: PriceQuoteParams): Promise<Pri
     ...params,
     buyToken: baseToken,
     sellToken: quoteToken,
+    isEthFlow: false,
```

Comment:
Isn't this derived from `buyToken === NATIVE_CURRENCY_BUY_ADDRESS` ?

Reply:
Looking at the code where that comes from, we also need to check whether it's a wrap.
Checking only for input currency address is not enough.
And that comes from a hook. Since we don't do hooks on smart components, it needs to be passed down.

I rather not change this at this point that I'm not that familiar with this part of the code yet